src/main/java/uk/gov/cshr/domain/Identity.java

```diff
@@ -24,10 +24,10 @@
 
     private boolean active;
 
-    @ManyToMany(cascade = CascadeType.ALL, mappedBy = "identities", fetch = FetchType.EAGER)
+    @ManyToMany(cascade = CascadeType.ALL,  fetch = FetchType.EAGER)
```

Comment:
Removing this mapping causes an error when deleting a role which has an identity attached

Reply:
But keeping it in means you can't update roles :D